wof/examples/flask/odm2/timeseries/odm2_config_timeseries.cfg

```diff
@@ -1,10 +1,11 @@
 [WOF]
 Network: ODM2Timeseries
 Vocabulary: ODM2Timeseries
+URLPATH: wofpysqlite
 Menu_Group_Name: ODM2
 # Change this to your configuration; assumes port 8080
-# http://serverip:port/networkcode/soap/cuahsi_1_0/.wsdl 
-Service_WSDL: http://127.0.0.1:8080/odm2timeseries/soap/cuahsi_1_0/.wsdl 
+# http://serverip:port/urlpath/soap/cuahsi_1_0/.wsdl
+Service_WSDL: http://127.0.0.1:8080/wofpysqlite/soap/cuahsi_1_0/.wsdl
```

Comment:
See my other comment on `test_config.cfg` about `Service_WSDL` now being obsolete (and even worse, if it's inconsistent with `URLPATH`).